### ISSUE TITLE
new columns memo and remaining stays empty

### ISSUE THREAD

**Issue by dheise-rheinenergie-trading-com** [cronicle-edge/cronicle-edge#27]
Hi Mike,

both new columns never show any value:

![image](https://user-images.githubusercontent.com/73586581/166662098-5fc21e79-353a-4d90-bd7f-723d0e51e4ff.png)

Can you explain how these columns work? Do I need to do anything else to get them to display anything?

**Comment by mikeTWC1984** [comment]
To get memo you need to print short message like below in your event:
echo "#test message#"

I'll put some samples in release notes

**Comment by dheise-rheinenergie-trading-com** [comment]
yeah! it works! Now I have to think about, where it could be useful for us.... :-P

**Comment by mikeTWC1984** [comment]
OK, I fixed all recent bug in 1.5.10. 
So that memo feature was mainly added for jobs that need to keep track of previous execution. E.g. you pulling some data from some data source, and you need to to know the offset where you stopped last time. In this scenario it also might be useful to indicate current offset once on the home page, rather than printing it to job log (generating useless large logs)

**Comment by dheise-rheinenergie-trading-com** [comment]
I installed 1.5.10 today. Column memo works fine but remaining still only shows "n/a". Any idea?

**Comment by mikeTWC1984** [comment]
OK, "remaining" only start populating after 10 seconds, which is by design of original cronicle ( I guess that makes sense). Try to watch some job that runs longer. I tried on my end and it works as expected.
